spring-cloud-zookeeper-core/src/main/java/org/springframework/cloud/zookeeper/ZookeeperProperties.java

```diff
@@ -69,6 +69,21 @@
 	 */
 	private TimeUnit blockUntilConnectedUnit = TimeUnit.SECONDS;
 
+	/**
+	 * The configured/negotiated session timeout in milliseconds.
+	 * Please refer to
+	 * <a href='https://cwiki.apache.org/confluence/display/CURATOR/TN14'>Curator's Tech
+	 * Note 14</a> to understand how Curator implements connection sessions.
+	 *
+	 * @see <a href='https://cwiki.apache.org/confluence/display/CURATOR/TN14'>Curator's Tech Note 14</a>
+	 */
+	private Integer sessionTimeoutMs = 60 * 1000;
```

Comment:
Change to `Duration` annotated with `@DurationUnit(milliseconds)` and drop `Ms` suffix. That way users can configure 1m or 90s or 10s or 200ms, etc... super nice. If they configure just 2000 without a suffix the annotation defaults to ms.